test/MockSmartStore.js

```diff
@@ -818,19 +818,99 @@ var MockSmartStore = (function(window) {
     return module;
 })(window);
 
-var mockStore = new MockSmartStore(false);
-var mockGlobalStore = new MockSmartStore(true);
-(function (cordova, store, globalStore) {
-    
+//var mockGlobalStore = new MockSmartStore(true);
+var globalStores = {};
+var userStores = {};
+
+var getStore = function (args) {
+   var isGlobal = args[0].isGlobalStore;
+   var storeName = args[0].storeName;
+   var store;
+
+   if(storeName == null)
+      storeName = "defaultStore";
```

Comment:
This has to match what's defined in `native`, correct? Can we remove this, so `native` is the single source of truth and we don't have to maintain them in both?